### DISCUSSION TITLE
[Q&A] Updating the GLPI Agent via a GLPI Deploy Task of a package?

### DISCUSSION THREAD

**Discussion by AndiMb** [glpi-project/glpi-agent#370] [Q&A]
### Documentation & bug reporting acknowledgment

Yes, I read it

### Describe your problem

Hi,

is there a way to update the GLPI Agent via a GLPI Deploy Task of a package? We are getting an error, when trying since the installation of the new GLPI Agent is terminating the run of the old GLPI Agent.

Regards,
Andreas

**Comment by g-bougard** [comment]
Hi @AndiMb
 
this is probably possible if you:
1. copy the MSI installer to a dedicated place
2. in place of start the install immediatly, use `schtasks` command to schedule the installation few minutes later so the current Deploy task as the time to be finished from the GLPI view.
